tests/test_resources/mpyl_config.yml

```diff
@@ -18,6 +18,7 @@ jira:
   userName: !ENV ${JIRA_USER_PASSWORD_USR:jira_user}
   password: !ENV ${JIRA_USER_PASSWORD_PSW:jira_password}
 docker:
+  envFileName: '.env'
```

Comment:
Remove this

Reply:
Ugh, I already [did](https://github.com/Vandebron/mpyl/pull/65/commits/e035025d6903b8c42159deae21b38ee6487cfa8a#diff-5c9c3ecafe366913e1c62262e90a186ed331e486791120777125d8bce648f7e4) so but all sorts of weird stuff hapened when I did a pull..